docs/conf.py

```diff
@@ -60,10 +60,11 @@
 exclude_patterns.append("_templates")
 exclude_patterns.append("_build")
 exclude_patterns.append("**.ipynb_checkpoints")
+exclude_patterns.append("**/ads.ipynb")
 
 # This is added to the end of RST files - a good place to put substitutions to
 # be used globally.
-rst_epilog += """
+rst_epilog = """
```

Comment:
This could be removed it not being used 🤔

Reply:
I think I'd like to keep it empty so the field is already there if somebody wants to add something.

Reply:
Keep everything, don't mess with conf.py!

Reply:
What's `ads.ipynb` ?

Reply:
The `+=` was throwing a warning as it had not been defined previously, hence changing it to `=`. And then `ads.ipynb` is a notebook that generates the list of papers that use TARDIS. It generates a page of documentation but we, at least for right now, do not want it in the documentation.

Reply:
I'm not sure if that's the appropriate way of not building that page. You can upload the rendered notebook and then nbsphinx does not build that notebook.

Reply:
I don't want sphinx to build it at all, not just nbsphinx

Reply:
Then use the full path to the file, without wildcards. Otherwise seems there are many files with that name.